packages/esm-patient-registration-app/translations/en.json

```diff
@@ -43,18 +43,19 @@
   "isDeadInputLabel": "Is Dead",
   "jumpTo": "Jump to",
   "maleLabelText": "Male",
-  "middleNameLabelText": "Middle Name (optional)",
+  "middleNameLabelText": "Middle Name",
   "months": "Months",
   "negativeMonths": "",
   "negativeYears": "",
   "no": "No",
   "numberInNameDubious": "",
+  "optional": "optional",
   "otherLabelText": "Other",
   "patient": "Patient",
-  "patientNameKnown": "Is Patient's Name Known?",
+  "patientNameKnown": "Patient's Name is Known?",
   "phoneEmailLabelText": "Phone, Email, etc.",
-  "phoneNumberInputLabelText": "Phone number (optional)",
-  "registerPatient": "Create Patient",
+  "phoneNumberInputLabelText": "Phone number",
+  "registerPatient": "Register Patient",
   "registrationSuccessToastDescription": "The patient can now be found by searching for them using their name or ID number",
   "registrationSuccessToastTitle": "New Patient Created",
```

Comment:
`registerPatient` changed from "Create Patient" to "Register Patient"... did you also want to update this one to be "New Patient Registered" ?

Reply:
No, let me check once again

Reply:
Hi @ZacButko!
There are a lot of translations which are to be fixed. I'll bring out a new PR by this week end. This includes providing a default value to all the translation keys.
Thanks!